rust/ql/lib/codeql/rust/security/SqlInjectionExtensions.qll

```diff
@@ -47,4 +48,9 @@ module SqlInjection {
   class SqlExecutionAsSink extends Sink {
     SqlExecutionAsSink() { this = any(SqlExecution e).getSql() }
   }
+
+  /** A sink for sql-injection from model data. */
+  private class ModelsAsDataSinks extends Sink {
+    ModelsAsDataSinks() { sinkNode(this, "sql-injection") }
+  }
```

Comment:
There's a design choice about whether we extend the query `Sink` class (as done here), or the `SqlExecution` concept itself in `Concepts.qll` - stemming from the fact that concepts and models-as-data have different philosophies that don't always mesh perfectly.  I've made the opposite choice in at least one other place, but I'm not convinced it was the right one.  Lets see how it goes and revisit to make all the sinks work the same way at some point in future.

Reply:
(I've created an issue internally to address this at some point; any thoughts expressed here will be picked up when I get to it)